build.gradle.kts

```diff
@@ -35,7 +35,12 @@ subprojects {
     apply(plugin = "nebula.kotlin")
 
     repositories {
-      jcenter()
+      jcenter() {
+        metadataSources {
+          artifact()
```

Comment:
This is required to pull `net.jodah:failsfe` which doesn't have a POM, just the artifact in jcenter.